### PR TITLE
Update QQ.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/QQ.com-Problematic.xml [hunk 1]
```diff
@@ -1,15 +1,102 @@
 <!--
 	For other rulesets of QQ.com, see QQ.xml
-
 	They don't have problem with https, just some problems are failing test.
 -->
-<ruleset name="QQ.com-Problematic">
-	<target host="qq.com" />
-	<target host="www.qq.com" />
+<ruleset name="QQ.com-Problematic" default_off='failed test'>
```
Comment (J0WI) [inline src/chrome/content/rules/QQ.com-Problematic.xml]: Please enable them by default, but [whitelist](https://github.com/EFForg/https-everywhere/blob/master/utils/ruleset-whitelist.csv) them from fetch tests.

File: src/chrome/content/rules/QQ.xml [hunk 2]
```diff
@@ -2,153 +2,123 @@
...
-		adsfile.qq.com	https://adsfile.qq.com/web/default0.swf
-		buy.qq.com
-		chuangshi.qq.com
-		img1.chuangshi.qq.com
-		sta1.chuangshi.qq.com
-		city.qq.com
-		cul.qq.com
-		data.qq.com
-		dldir2.qq.com
-		dlied6.qq.com
```
Comment (J0WI) [inline src/chrome/content/rules/QQ.xml]: Please keep this host

File: src/chrome/content/rules/QQ.xml [hunk 3]
```diff
@@ -2,153 +2,123 @@
...
-		gameweb-img.qq.com
-		gongyi.qq.com       Akamai
-		live.qq.com
-		img1.qq.com
-		iwan.qq.com
-		open.mail.qq.com
-		shangjia.map.qq.com
-		mat1.qq.com
-		mil.qq.com
-		mx.qq.com
```
Comment (J0WI) [inline src/chrome/content/rules/QQ.xml]: Please keep this host

File: src/chrome/content/rules/QQ.com-Problematic.xml (hunk 1 above)
Comment (ivysrono) [reply src/chrome/content/rules/QQ.com-Problematic.xml]: Done. Thanks.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**